ocaml/xapi/xapi_xenops.ml

```diff
@@ -718,6 +718,7 @@ module MD = struct
 
   let vusbs_of_vm ~__context (vmref, vm) =
     vm.API.vM_VUSBs
+    |> List.filter (fun self -> Db.VUSB.get_currently_attached ~__context ~self)
```

Comment:
As there are two types DB query of VUSB, I think we could use `Db.VUSB.get_record` first so that we could decrease the DB query amount.

Reply:
Thanks @krizex. Good point. Updated.